packages/vite/src/node/plugins/manifest.ts

```diff
@@ -103,7 +103,7 @@ export function manifestPlugin(config: ResolvedConfig): Plugin {
       function createAsset(chunk: OutputAsset): ManifestChunk {
         const manifestChunk: ManifestChunk = {
           file: chunk.fileName,
-          src: chunk.name
+          src: normalizePath(chunk.name!)
         }
 
         if (cssEntryFiles.has(chunk.name!)) manifestChunk.isEntry = true
```

Comment:
I'm unsure if this reference to `chunk.name` also needs to be normalised.

Reply:
Thank you for the PR!

I think it is better to normalize the name here. (`cssFileName`)
https://github.com/vitejs/vite/blob/ccb3449edcbe08948a2d01e5d37dbca0d12b582a/packages/vite/src/node/plugins/css.ts#L536-L545

If you don't mind, I will push some changes since I'm using windows. 🙂

Reply:
Of course not! Would love any input and help with this one.